.gitignore

```diff
@@ -19,3 +19,5 @@ pip-wheel-metadata
 .vscode
 Pipfile
 Pipfile.lock
+AUTHORS
```

Comment:
This may be because I didn't re-evaluate after merging with recent changes; but I built RPMs several times, and git was complaining about AUTHORS and ChangeLog so I just added them. I guess I'll try again and confirm... and... yeah, I deleted `AUTHORS` and `ChangeLog` and built a new COPR server RPM. The files failed to reappear, so I'll drop these back out.